makefiles/boards/alias.inc.mk

```diff
@@ -0,0 +1,10 @@
+# list of known alias
+ALIAS.phynode   = pba-d-01-kw2x
+#TODO remove this line (as its just for show)
+ALIAS.linux     = native
+
+# this temporary variable is needed as we can not use BOARD when assigning BOARD
+ALIAS_TARGET := $(ALIAS.$(BOARD))
```

Comment:
`ALIAS_TARGET` is just used in here but is a generic name, I would like to see it namespaced, and so use `BOARD_ALIAS_TARGET`, or just `BOARD_ALIAS`.